### PR TITLE
Race war now, aka species branch changes and cleanup

### PR DESCRIPTION
**Pull request by Roland410**
🆑Roland410
tweak:Akula, Sergals, Vasilissan and Vulpkanin were merged into the genemodder race.
tweak:Prometheans and Dionaea can once again be full members of NTEF.
tweak:Prometheans are no longer allowed to play SMC.
tweak:Plasmemes were kicked out and can now only be civilians or Solgov Scientists.
tweak:Adherents can now play all civilian jobs except a few such as Adjudicator, Bailiff etc.
tweak:Most species can fully play as the Solgov branch where applicable, this means SGR (currently only the lawyer), SFP (Forensics Tech) and Solgov Scientist (scientist).
tweak:Species who can play SFP can also play SMC now.
bugfix:Brings the dionaea age points in line with the already existing ones.
/🆑
If I missed anything let me know, but this should be good for merging unless anyone has issues with it, but this was mostly spoken to Weezer about, and Stutter helped me with the rest.

### CODE REVIEW

**Comment by Roland410** [comment]
image
Weezer should be leaving a comment himself soon, but until then, have this screencap.

**Comment by Sparrowbird611** [comment]
https://cdn.discordapp.com/attachments/529565613123829780/707020433597399102/THE_RACE_WAR.mp4

THE RACE WAR STARTS NOOOOOOOOW!!

**Comment by John-Dang** [comment]
This update has my seal of lore approval, citizenship for aliens time.

**Comment by Tename** [comment]
Again, this is going to interfere majorly with peoples existing characters and there isn't really a super big reason to merge the 3 species into genemodder, it's just going to result in more snowflakey named races because there isn't a set option.

**Comment by Roland410** [comment]
This was already approved by Weezer, after the staff discussed it. I'm only solving the issues someone introduced when they ported it like this anyways. You can still have a custom species name and be a Sergal, etc. and even look like one, but code-wise it'll be under the genemodder race.

**Comment by Tename** [comment]
What's the issue there is no issue. You are only creating problems for the people who play these races for an alleged issue no one complains about.

**Comment by Roland410** [comment]
It's a code quality issue, which wouldn't be one, if it was ported in a sane manner. What you're talking about is the consequence of it.
These races have next to no (and for most of them, zero) differences from humans from a code perspective, and the existence of the custom species/genemodder race should have allowed everyone to make them.
As I said above, this was also approved by staff already.

**Comment by Tename** [comment]
They have their own set lore and dedicated names, without it there is nothing encouraging having a specific species name or following the already existing lore. Why are you not merging together skrell then, they have little to no differences from humans. This is such a minor and petty issue to fuck over the people who play these species.

**Comment by Roland410** [comment]
You can literally still play these species. You are making an issue out of something that is not one. If their lore would be an issue, the lore headdev/maintainer (Weezer) wouldn't have approved it either. This was already discussed before I made the PR and each part of it specifically confirmed with Weezer and Stutter (the headmin) personally.

File: maps/torch/job/torch_jobs.dm [hunk 1]
```diff
@@ -11,8 +8,9 @@
 	)
 
 	species_to_job_blacklist = list(
+		/datum/species/adherent = list(/datum/job/psiadvisor, /datum/job/liaison, /datum/job/bodyguard, /datum/job/adjudicator, /datum/job/bailiff, /datum/job/detective),
```
Comment (Superhats) [inline maps/torch/job/torch_jobs.dm:11]: Would that mean adherents can now be in security?

Comment (Roland410) [reply maps/torch/job/torch_jobs.dm:11]: Nope, they are locked to civilian branches and blacklisted detective, so it shouldn't change much. They can however play chaplain now, but iirc that's the only change, as I've blacklisted the civvie jobs they shouldn't be having.

**Comment by Tristan63** [comment]
the fact that trees are being allowed to exist deserves a thumbs down

**Pull request merged**